source/install.txt

```diff
@@ -17,6 +17,8 @@ Prerequisites
 
 .. include:: /includes/fact-connect-prereqs.rst
 
+.. include:: /includes/platform-limitations.rst
```

Comment:
Thanks @jason-price-mongodb. Are you satisfied with this as a top level heading our would you consider this a subheading, similar to the Supported MDB Versions. I see it as the later, but am fairly agnostic.

Reply:
I think it should be left as is because "Supported MDB Versions" implies it covers all supported versions and configurations, which it probably doesn't and would therefore be technically inaccurate now and over time in future releases.